### PR TITLE
Rebased: Selectively run or ignore tests with --grep and --invert

### PR DESCRIPTION
**Pull request by keyscores**
@aldeed this replaces PR: https://github.com/DispatchMe/meteor-mocha/pull/14
It is not rebased and includes your suggested syntax.

Fixes https://github.com/DispatchMe/meteor-mocha/issues/5 and https://github.com/DispatchMe/meteor-mocha/issues/7

### CODE REVIEW

**Comment by keyscores** [comment]
@aldeed any thoughts on this? I think all your requested changes are here.

**Comment by keyscores** [comment]
@aldeed ping, I think all the changes requested are here.

**Comment by keyscores** [comment]
@aldeed  ping

**Comment by keyscores** [comment]
@aldeed I guess you're busy on other projects. Or is there any reason this should not be merged? Thanks

**Comment by thebarty** [comment]
PLUS one for giving some write access to keyscores

File: client.js [hunk 1]
```diff
@@ -1,14 +1,19 @@
...
 function runTests() {
   // We need to set the reporter when the tests actually run. This ensures that the
   // correct reporter is used in the case where `dispatch:mocha-browser` is also
   // added to the app. Since both are testOnly packages, top-level client code in both
   // will run, potentially changing the reporter.
-  mocha.reporter(Meteor.settings.public.CLIENT_TEST_REPORTER || 'spec');
+  const { mochaOptions } = Meteor.settings.public.runtimeArgs || {};
+  const { clientReporter, grep, invert, reporter } = mochaOptions || {};
+  if (grep) mocha.grep(grep);
+  if (invert) mocha.options.invert = invert;
```
Comment (aldeed) [inline client.js]: It looks like the property is actually called `grepInvert`

**Comment by aldeed** [comment]
@keyscores I noticed one issue, but I'll just merge this and fix it. I'd like to give write access but I need to get approval from Dispatch first since this is their repo. I'll see what I can do.

**Pull request merged**